### ISSUE TITLE
Unknown Error

### ISSUE THREAD

**Issue by Abhishekzysk** [VemtoOrg/vemto2-issues#122]
**Error Message:**
Unknown Error

**Stack:**
```
schema-reader-db Error: #0 phar://C:/Users/Abhishek M/AppData/Local/Programs/Vemto/resources/static/VMTTL2/index.php(87): VemtoDBReader\DatabaseManager->createDatabase('vemto_schema_re...')
#1 phar://C:/Users/Abhishek M/AppData/Local/Programs/Vemto/resources/static/VMTTL2/common/Vemto.php(111): {closure}()
#2 phar://C:/Users/Abhishek M/AppData/Local/Programs/Vemto/resources/static/VMTTL2/index.php(18): Vemto\Vemto::execute('schema-reader-d...', Object(Closure))
#3 C:\Users\Abhishek M\AppData\Local\Programs\Vemto\resources\static\VMTTL2(14): require('phar://C:/Users...')
#4 {main}
```

---

Vemto 2.0.9
win32 10.0.19045
Locale: en-US
App version: 2.0.9

**Comment by vimitali368** [comment]
Try to upgrade to version 2.0.10, I have the same error gone.

**Comment by TiagoSilvaPereira** [comment]
@vimitali368 thanks for your help by reporting 2.0.10 solves this issue. 

@Abhishekzysk please also check if all dependencies for Vemto are installed:

![image](https://github.com/TiagoSilvaPereira/vemto2-issues/assets/11933789/bb0e2bf2-d33a-475e-8377-c525462e7b41)

**Comment by TiagoSilvaPereira** [comment]
Fixed, please upgrade to 2.0.11
